tests/integration/targets/manager_test/tasks/main.yml

```diff
@@ -0,0 +1,60 @@
+- name: Create session
+  redfish_command:
+    category: Sessions
+    command: CreateSession
```

Comment:
This command is not supported by this module.

Reply:
The command list of redfish_command is as follows -

CATEGORY_COMMANDS_ALL = {
    "Systems": ["PowerOn", "PowerForceOff", "PowerForceRestart", "PowerGracefulRestart",
                "PowerGracefulShutdown", "PowerReboot", "SetOneTimeBoot", "EnableContinuousBootOverride", "DisableBootOverride"],
    "Chassis": ["IndicatorLedOn", "IndicatorLedOff", "IndicatorLedBlink"],
    "Accounts": ["AddUser", "EnableUser", "DeleteUser", "DisableUser",
                 "UpdateUserRole", "UpdateUserPassword", "UpdateUserName",
                 "UpdateAccountServiceProperties"],
    "Sessions": ["ClearSessions", "CreateSession", "DeleteSession"],
    "Manager": ["GracefulRestart", "ClearLogs", "VirtualMediaInsert",
                "VirtualMediaEject", "PowerOn", "PowerForceOff", "PowerForceRestart",
                "PowerGracefulRestart", "PowerGracefulShutdown", "PowerReboot"],
    "Update": ["SimpleUpdate"]
}

This dictionary contains CreateSession.

Reply:
Works fine with latest module

Reply:
I believe that one is inherited from the parent class.

Reply:
Sorry, I looked at the wrong module.